apps/web/screens/Syslumenn/Auctions.tsx

```diff
@@ -0,0 +1,330 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useEffect, useState } from 'react'
+import {
+  Box,
+  FocusableBox,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  NavigationItem,
+  Option,
+  Select,
+  Tag,
+  Text,
+} from '@island.is/island-ui/core'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import {
+  ContentLanguage,
+  Query,
+  QueryGetAuctionsArgs,
+  QueryGetNamespaceArgs,
+  QueryGetOrganizationPageArgs,
+} from '@island.is/web/graphql/schema'
+import { GET_NAMESPACE_QUERY, GET_ORGANIZATION_PAGE_QUERY } from '../queries'
+import { Screen } from '../../types'
+import { useNamespace } from '@island.is/web/hooks'
+import { useLinkResolver } from '@island.is/web/hooks/useLinkResolver'
+import { OrganizationWrapper } from '@island.is/web/components'
+import { CustomNextError } from '@island.is/web/units/errors'
+import getConfig from 'next/config'
+import { GET_AUCTIONS_QUERY } from '@island.is/web/screens/queries/Auction'
+import { useQuery } from '@apollo/client'
+import { useDateUtils } from '@island.is/web/i18n/useDateUtils'
+import { useRouter } from 'next/router'
+
+const { publicRuntimeConfig } = getConfig()
+
+interface AuctionsProps {
+  organizationPage: Query['getOrganizationPage']
+  namespace: Query['getNamespace']
+}
+
+const Auctions: Screen<AuctionsProps> = ({ organizationPage, namespace }) => {
+  const { disableSyslumennPage: disablePage } = publicRuntimeConfig
+  if (disablePage === 'true') {
+    throw new CustomNextError(404, 'Not found')
+  }
+
+  const n = useNamespace(namespace)
+  const { linkResolver } = useLinkResolver()
+  const { format } = useDateUtils()
+  const Router = useRouter()
+
+  const pageUrl = `/stofnanir/syslumenn/uppbod`
```

Comment:
Changed it so that we get the url from the Router. 

Also, all the `navList` code (and the syslumenn screens in general) will need to be refactored at some point as there is a lot of duplicate code.